src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -63,14 +65,15 @@ public Command fastOuttakeGamepiece() {
         },
         () -> {
           set(IntakeConstants.OUTTAKE_REST_SPEED);
-        });
+        },
+        this);
   }
 
   public Command autoOuttakeGamepiece() {
-    return Commands.runOnce(() -> set(IntakeConstants.OUTTAKE_SPEED));
+    return Commands.runOnce(() -> set(IntakeConstants.OUTTAKE_SPEED), this);
```

Comment:
```suggestion
    return this.runOnce(() -> set(IntakeConstants.OUTTAKE_SPEED), this);
```

Other places in the file too